vm/systemSmartContracts/auction.go

```diff
@@ -691,7 +732,7 @@ func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.Retu
 				}
 				continue
 			}
-
+			//TODO: check this: too late to check maxStakePerNode
```

Comment:
it is not too late - this is an optional argument - so if the user makes something wrong the totalStakeValue is used as maxStakePerNode - most of the validators will not bother setting the maxStakePerNode

Reply:
ok